lib/recurly/frame.js

```diff
@@ -46,7 +46,7 @@ class Frame extends Emitter {
   constructor ({ recurly, path, payload = {}, ...options }) {
     super();
     this.recurly = recurly;
-    this.id = `${this.recurly.id.split('-')[0]}-${uuid().split('-')[0]}`;
+    this.id = `${this.recurly.id}-${uid()}`;
```

Comment:
No need to parse these anymore since our nanoid charset does not include hyphens.